docs/spec.md

```diff
@@ -0,0 +1,96 @@
+# Operational Data Standard
+
+Standardized data for providers of scheduled mobility services.
+
+## Overview
+
+Providers of scheduled transit service have scheduling needs that extend beyond the capabilities of the General Transit Feed Specification (GTFS), which was designed to meet the needs of the public. Creating, maintaining, updating, and implementing a transit schedule requires providers to have a more complete view of daily vehicle operations and personnel schedules. The Operational Data Standard (ODS) is a proposal for a new open standard to define deadheads and runs. ODS is based on, and ties back to, GTFS-static files. ODS is proposed as a standard separate from GTFS in order to preserve the privacy of internal operations data.
+
+## Deadheads
+
+Deadheads are vehicle movements during which a transit vehicle is not in service. The Deadheads module of ODS contains three new files in addition to the files in GTFS. This module provides for the creation of scheduled deadheads tied to an existing GTFS feed and linked to significant locations (such as points where pull in, pull out, and layovers occur) in the service area.
+
+| **File Name** | **Description** |
+| --- | --- |
+| deadheads.txt | Defines scheduled deadheads contained in a transit feed. (This file is analogous to trips.txt for non-revenue operations.) |
+| ops_locations.txt | Significant operational locations relevant to the performance of vehicle deadheads. (This file is analogous to stops.txt for non-revenue operations.) |
+| deadhead_times.txt | Times that a vehicle arrives at and departs from operational locations for each deadhead. (This file is analogous to stop_times.txt for non-revenue operations.) |
+
+### deadheads.txt
+
+| **Field Name** | **Type** | **Required** | **Description** |
+| --- | --- | --- | --- |
+| deadhead_id | ID | Required | Identifies a deadhead. |
+| service_id | ID referencing **calendar.service_id** | Required | Identifies a set of dates when the deadhead is scheduled to take place. |
```

Comment:
Does it make sense to link to the GTFS spec/files where we reference them, such as here?